app/dashboard/migrations/0069_auto_20180507_1841.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 2.0.4 on 2018-05-07 18:41
+
+from django.db import migrations
+
+
+def forwards_func(apps, schema_editor):
+    Profile = apps.get_model('dashboard', 'Profile')
+    profiles = Profile.objects.filter(github_access_token='').all()
```

Comment:
We'll want to make sure we're not marking valid profiles to hidden if they have a `user` as well.  Also, we should be able to just do a `Profiles.objects.filter(github_access_token='').update(hide_profile=True)`

Reply:
3d7e03ad8996c3057490b56d8846313f09af67a8